File: Services/Document/SimpleERP.Document.API/Infrastructure/Models/DocumentInfoModel.cs
using SimpleERP.Libraries.Infrastructure.Entities;
using SimpleERP.Libraries.Infrastructure.Excel;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SimpleERP.Document.API.Infrastructure.Models
{
    public class DocumentInfoModel
    {
        [ExcelProperty]
        public long? Id { get; set; }
        [ExcelProperty]
        public string  No { get; set; }
        [ExcelProperty]
        public string Subject { get; set; }
        public long[] DocumetFileIds { get; set; }
        [ExcelProperty]
        public string Text { get; set; }
        [ExcelProperty]
        public string DateOfRelease { get; set; }
        [ExcelProperty]
        public string DateOfCreate { get; set; }
        [ExcelProperty]
        public string Creator { get; set; }
        public string DateOfModify { get; set; }
        public string Modifier { get; set; }
        public long IssuerId { get; set; }
        [ExcelProperty]
        public string IssuerTitle { get; set; }
        public long DomainId { get; set; }
        [ExcelProperty]
        public string DomainTitle { get; set; }
        public long TypeId { get; set; }
        [ExcelProperty]
        public string TypeTitle { get; set; }
    }
}

File: UI/SimpleERP.UI.Web/ClientApp/src/app/modules/document/models/documentinfo.model.ts
export interface IDocumentInfoModel {
  id?: number;
  no: string;
  subject: string;
  text: string;
  dateOfRelease: string;
  dateOfCreate: string;
  creator: string;
  dateOfModify: string;
  modifier: string;
  issuerId: number;
  issuerTitle: string;
  domainId: number;
  domainTitle: string;
  typeId: number;
  typeTitle: string;
  DocumetFileIds: number[];
}


export class DocumentInfoModel implements IDocumentInfoModel {
  constructor(
    public id: number | null,
    public no: string,
    public subject: string,
    public text: string,
    public dateOfRelease: string,
    public dateOfCreate: string,
    public creator: string,
    public dateOfModify: string,
    public modifier: string,
    public issuerId: number,
    public issuerTitle: string,
    public domainId: number,
    public domainTitle: string,
    public typeId: number,
    public typeTitle: string,
    public DocumetFileIds: number[]
  ) { }
}
